hazelcast/src/main/resources/hazelcast-default.xml

```diff
@@ -197,11 +199,6 @@
         <value-collection-type>SET</value-collection-type>
     </multimap>
 
-    <multimap name="default">
-        <backup-count>1</backup-count>
-        <value-collection-type>SET</value-collection-type>
-    </multimap>
-
```

Comment:
That is deleted so I expect it is not used anymore and including it will not validate the xml.